packages/pulumi-aws/src/apps/api/ApiPageBuilder.ts

```diff
@@ -0,0 +1,462 @@
+import * as path from "path";
+import * as pulumi from "@pulumi/pulumi";
+import * as aws from "@pulumi/aws";
+
+//@ts-ignore
+import { createInstallationZip } from "@webiny/api-page-builder/installation";
+import { PulumiApp } from "@webiny/pulumi-sdk";
+import { Vpc } from "./ApiVpc";
+
+interface PageBuilderParams {
+    env: Record<string, any>;
+    primaryDynamodbTableArn: pulumi.Input<string>;
+    fileManagerBucketId: pulumi.Input<string>;
+    cognitoUserPoolArn: pulumi.Input<string>;
+    awsAccountId: pulumi.Input<string>;
+    awsRegion: pulumi.Input<string>;
+    vpc: Vpc | undefined;
+}
+
+export function createPageBuilder(app: PulumiApp, params: PageBuilderParams) {
+    app.addHandler(() => {
+        const pbInstallationZipPath = path.join(path.resolve(), ".tmp", "pbInstallation.zip");
+        // Will create "pbInstallation.zip" and save it in the `pbInstallationZipPath` path.
+        createInstallationZip(pbInstallationZipPath);
+
+        new aws.s3.BucketObject("./pbInstallation.zip", {
+            key: "pbInstallation.zip",
+            acl: "public-read",
+            bucket: params.fileManagerBucketId,
+            contentType: "application/octet-stream",
+            source: new pulumi.asset.FileAsset(pbInstallationZipPath)
+        });
+    });
```

Comment:
@kedrzu does this mean that it will be creating and uploading this zip on every deploy?

Reply:
Probably - that's basically a rip off from here: https://github.com/webiny/webiny-js/blob/next/api/pulumi/dev/pageBuilder.ts#L35

Reply:
Ah got it, so it's the same as it currently is. That's cool. We'll revisit it later. Thanks.